EIPS/eip-5920.md

```diff
@@ -45,6 +45,7 @@ A new opcode is introduced: `PAY` (`0xfc`), which:
 - Pops two values from the stack: `addr` then `val`.
 - Charges the gas cost detailed below.
 - Exceptionally halts if `addr` has any of the high 12 bytes set to a non-zero value (i.e. it does not contain a 20-byte address).
+- Exceptionally halts if `val` is nonzero if the `STATIC` flag of the current call frame is `true`
```

Comment:
I don't think we should complicate it with wether `val` is used or not.  There are better non-writing ways to "warm" a target address.

```suggestion
- halt with exceptional failure if the current frame is in `static-mode`.
```

Reply:
Ok, this is also fine by me. I copied the behavior of call, which allows calls if no value is sent (this is of course used in static contexts to call libraries for instance). I also used `STATIC` flag which is the original EIP terminology. But I think `static-mode` is also clear for every EVM developer.

Reply:
Actually, indeed, also for the warm use case: `BALANCE` is clearly used to read, and `PAY` is clearly used to write. So `PAY` should not be used in a static context.